server/routes/visitJourney/visitType.test.ts

```diff
@@ -36,9 +26,7 @@ afterEach(() => {
   jest.resetAllMocks()
 })
 
-describe.skip('/visit/:reference/update/visit-type', () => {
```

Comment:
Tests that were here for update journey were skipped. Replaced with the book journey tests; will add testing for update journey routes in subsequent PR